### DISCUSSION TITLE
Can't see more than default number of languages

### DISCUSSION THREAD

**Discussion by Toma400** [anuraghazra/github-readme-stats#3207] [Q&A] [answered]
### Describe the bug

I tried making this work for a while, but now visiting this place again, I see it hasn't been changed at all.  
In short, setting up my language counter, it does not take `&langs_count=` parameter, no matter how I write it. In example code like this: [https://github-readme-stats-sigma-five.vercel.app/api/top-langs/?username=toma400&langs_count=8&count_private=true&theme=merko&border_color=248610](https://github-readme-stats-sigma-five.vercel.app/api/top-langs/?username=toma400&langs_count=8&count_private=true&theme=merko&border_color=248610) we can see there's only 5 languages shown, but the count is using 8 (I took this exactly from example).
And in case this isn't obvious, but there's at least 1 language that should be shown after HTML, being Ruby, and I assume there'd be a bit of Kotlin as well, since private repos are counted.

### Expected behavior

Correct showcasing of languages amount depending on `&langs_count=` parameter.

### Screenshots / Live demo link

![ucl](https://github.com/anuraghazra/github-readme-stats/assets/65849429/d1cc005d-b7e8-4b29-9fa5-08492cdceb14)

**Comment by rickstaa** [comment, marked as answer]
@Toma400, thanks for your question. It looks like you are using an outdated Vercel instance. Please use the main Vercel instance (i.e. `https://github-readme-stats.vercel.app`) or better [deploy your own instance](https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#deploy-on-your-own). 

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=toma400&langs_count=12&count_private=true&theme=merko&border_color=248610)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=toma400&langs_count=12&count_private=true&theme=merko&border_color=248610)](https://github.com/anuraghazra/github-readme-stats)
```

**Reply by Toma400** [reply]
Oh, I see, this indeed works! Thank you for answering the question <3 I assume deploying your own let you have more up-to-date statistics without putting weight of call on main instance used by everyone else?

**Reply by rickstaa** [reply]
Correct! For your instance, you can remove the rate limit (see https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#disable-rate-limit-protections) ⚡.

**Reply by rickstaa** [reply]
To keep your instance up to date see https://github.com/anuraghazra/github-readme-stats?tab=readme-ov-file#keep-your-fork-up-to-date.
